### PR TITLE
Use regex to parse link-header

### PR DESCRIPTION
**Pull request by billbogaiv**
New behavior will extract value, but assumes exact format. Might make sense to be more lenient with parsing.
Needs more testing...

### CODE REVIEW

File: src/Site/Infrastructure/GitHub/Release.cs [hunk 1]
```diff
@@ -36,10 +37,10 @@ public ReleasesResponse ParsePaging(string header)
                 {
                     var values = x.Split(';');
```
Comment (kendaleiv) [inline src/Site/Infrastructure/GitHub/Release.cs:38]: This `value` variable could also be removed if it's OK to remove the `Console.WriteLine` call too.

File: src/Site/Infrastructure/GitHub/Release.cs [hunk 2]
```diff
@@ -36,10 +37,10 @@ public ReleasesResponse ParsePaging(string header)
...
                     Console.WriteLine(values[0]);
-                    var uri = new Uri(values[0].Replace("<", "").Replace(">", ""));
+                    var uri = new Uri(Regex.Match(values[0], "<(.*?)>").Groups[1].Value);
                     var queryDictionary = QueryHelpers.ParseQuery(uri.Query);
```
Comment (kendaleiv) [inline src/Site/Infrastructure/GitHub/Release.cs]: This `queryDictionary` variable could also be removed.

**Comment by kendaleiv** [comment]
What's the rationale for this change? If it's performance related we probably want to set **[RegexOptions](https://msdn.microsoft.com/en-us/library/system.text.regularexpressions.regexoptions%28v=vs.110%29.aspx).Compiled** too.

**Comment by billbogaiv** [comment]
@kendaleiv: rebased to include `Compiled`-option.

**Pull request merged**